mikrotik/provider_test.go

```diff
@@ -25,7 +27,7 @@ func testAccPreCheck(t *testing.T) {
 	if v := os.Getenv("MIKROTIK_USER"); v == "" {
 		t.Fatal("The MIKROTIK_USER environment variable must be set")
 	}
-	if v := os.Getenv("MIKROTIK_PASSWORD"); v == "" {
+	if _, exists := os.LookupEnv("MIKROTIK_PASSWORD"); !exists {
 		t.Fatal("The MIKROTIK_PASSWORD environment variable must be set")
 	}
```

Comment:
The main reason to use a `LookupEnv` is to allow using an account without a password (i.e., the value of the environment variable is an empty string). It checks if `MIKROTIK_PASSWORD` environment variable exists and doesn't throw an exception if it's empty.

Happy to switch it back to what it was for consistency, but it may be worth updating the provider to use `LookupEnv` too.

Reply:
I checked the rest of the code, and it looks that the behaviour is consistent. There is a check if any of the environment variables is empty in `client/client.go`, but it does nothing in that case:

https://github.com/ddelnano/terraform-provider-mikrotik/blob/d9648a2829c4c7264195607bffbfcd2afaf94464/client/client.go#L173-L175

There are no other places where it'd be used differently.